### PR TITLE
Wilders talk

### PR DESCRIPTION
**Pull request by hyanwong**
This requires the ability to show vimeo videos. I think this is roughly right, but I guess @benjeffery will want to check it over.

### CODE REVIEW

File: _includes/video.html [hunk 1]
```diff
@@ -1,7 +1,13 @@
 <div class="video">
     {{ video.title }}
     <iframe width="560" height="315"
+        {% if video.youtube-id %}
         src="https://www.youtube-nocookie.com/embed/{{ video.youtube-id}}"
+        {% elsif video.vimeo-id %}
+        src="https://www.youtube-nocookie.com/embed/{{ video.youtube-id}}"
```
Comment (jeromekelleher) [inline _includes/video.html:7]: shouldn't that be vimeo-id?

**Review by jeromekelleher — COMMENTED** [review]
LGTM, but yeah let's get Ben's looksee first.

File: _includes/video.html (hunk 1 above)
Comment (hyanwong) [reply _includes/video.html:7]: Oh, good spot - copy/paster error. Will fix

**Comment by benjeffery** [comment]
I've added a commit with support for vimeo - annoyingly I have had to make it a direct link, not an embed as vimeo doesn't support the embed for wilders video without a code only avliable to the video author. Vimeo also doesn't supply thumbnail URLs without an API request, so I've done that manually.
Sorry @hyanwong, the video file you edited was unused (from the old site) I forgot to remove it, but have done now.

**Comment by hyanwong** [comment]
>  vimeo doesn't support the embed for wilders video without a code only avliable to the video author. 

Shall I ask Wilder if he can get an embed link?

**Pull request merged**